remote_branch_checker/remote_branch_checker.sh

```diff
@@ -405,9 +407,13 @@ ${phpcmd} ${mydir}/../../moodlecheck/cli/moodlecheck.php \
 # Exclude build directories from the results (e.g. lib/yui/build, lib/amd/build/)
 find $WORKSPACE -type d -path \*/build | sed "s|$WORKSPACE/||" > $WORKSPACE/.jshintignore
 
-# Run the JSHINT (using the checked out .jshint file)
-${jshintcmd} --config $WORKSPACE/.jshintrc --exclude-path $WORKSPACE/.jshintignore \
-    --reporter=checkstyle ${WORKSPACE} > "${WORKSPACE}/work/jshint.xml"
+if [ -f $WORKSPACE/.eslintrc ]; then
+    ./node_modules/.bin/eslint -f checkstyle $WORKSPACE > "${WORKSPACE}/work/eslint.xml"
```

Comment:
This raw use of node_modules might be controversial, thoughts welcome. (I think we should use the grunt installed version because we then don't need to worry about version dependencies.